site/en/docs/extensions/mv3/manifest/homepage_url/index.md

```diff
@@ -2,14 +2,18 @@
 layout: "layouts/doc-post.njk"
 title: "Manifest - Homepage Url"
 date: 2013-05-12
-updated: 2018-04-26
+updated: 2023-01-04
 description: Reference documentation for the homepage_url property of manifest.json.
 ---
 
-The URL of the homepage for this extension. The extensions management page (chrome://extensions)
-will contain a link to this URL. This field is particularly useful if you [host the extension on
-your own site][1]. If you distribute your extension using the [Chrome Web Store][2], the homepage
-URL defaults to the extension's own page.
+An optional Manifest key containing a String for a valid homepage URL. Developers may choose to set the extension's homepage to their personal or company's website. If the parameter is left undefined, the default homepage will be the extension's Chrome Web Store page listed on the extensions management page (chrome://extensions). This field is particularly useful if you [host the extension on your own site][1]. 
```

Comment:
I'm working on style guidance for Extension-specific features. In general, manifest and string should NOT be capitalized. They are common nouns because they refer to classes of things rather than instances of things. It's the same as with 'I live in a city' vs. 'I live in Kansas City'.

Also, I should have made it clear yesterday that the kind of review we're asking for is a review of the technical accuracy of the PR. One of the writers will review for typos when you're done.